vcpkg/vcpkg.json

```diff
@@ -0,0 +1,28 @@
+{
+  "description": "A C++ library for working with the Bethesda archive file format",
+  "homepage": "https://github.com/Ryan-rsm-McKenzie/bsa",
+  "documentation": "https://ryan-rsm-mckenzie.github.io/bsa/",
+  "supports": "x64 & (windows | linux) & static",
+  "name": "bsa",
+  "version-semver": "2.0.2",
+  "dependencies": [
+    "lz4",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    },
+    "zlib"
+  ],
+  "features": {
+    "tests": {
+      "description": "Build tests",
+      "dependencies": [
+        "catch2"
+      ]
+    }
+  }

```

Comment:
```suggestion
```
The tests are unnecessary for consumers. Vcpkg would just end up building them and then throwing away the resulting binary without actually running it.